TouchPortal-API/TouchPortalAPI/__init__.py

```diff
@@ -42,14 +42,15 @@ class Client(ExecutorEventEmitter):
     SND_BUFFER_SZ = 32**4  # [B] maximum size of send data buffer (1MB)
     SOCK_EVENT_TO = 1.0    # [s] timeout for selector.select() event monitor
 
-    def __init__(self, pluginId, sleepPeriod=0.01, autoClose=False, checkPluginId=True, maxWorkers=None, executor=None):
+    def __init__(self, pluginId, sleepPeriod=0.01, autoClose=False, checkPluginId=True, updateStatesOnBroadcast=True, maxWorkers=None, executor=None):
```

Comment:
Wait... default of `True`?  Is that intentional? This would fire on every page change...

Reply:
Also it would be useful to add the new attrib to the class's docstring with a short explanation of what it does.

Reply:
as explained [here](https://github.com/KillerBOSS2019/TouchPortal-API/pull/11#discussion_r667928626) this is indeed intentional.
I indeed need to update the doc string